### PR TITLE
Update Get-CsOnlineUser.md

### PR DESCRIPTION
**Pull request by isrumnon**

### CODE REVIEW

**Comment by isrumnon** [comment]
Hi @yogkumgit can you please get these changes merged asap please? It's urgent

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 1]
```diff
@@ -497,6 +498,13 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -Sort/-OrderBy
```
Comment (get-itips) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:501]: We can't use more than one parameter name here, please split in two and also add the yaml section to conform with cmdlet's article format

Comment (isrumnon) [reply skype/skype-ps/skype/Get-CsOnlineUser.md]: @get-itips I've modified it. Thanks!

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 2]
```diff
@@ -211,9 +212,9 @@ Enables you to limit the returned data by filtering on specific attributes. For
 
 The Filter parameter uses the same filtering syntax as the Where-Object cmdlet. For example, the following filter returns only users who have been enabled for Enterprise Voice: `-Filter 'EnterpriseVoiceEnabled -eq $True'` or ``-Filter "EnterpriseVoiceEnabled -eq `$True"``.
 
-**Updates in Teams PowerShell Module version 5.0.0**
+**Updates in Teams PowerShell Module version 5.0.0 and above**
 
-The following updates are applicable for organizations having TeamsOnly users that use Microsoft Teams PowerShell version 5.0.0  (note that these changes are only rolled out in commercial environments at present):
+The following updates are applicable for organizations having TeamsOnly users that use Microsoft Teams PowerShell versions >=5.0.0 and will be applicable to all Teams PowerShell Module versions by 15th April 2023 (note that these changes are only rolled out in commercial environments at present):
```
Comment (dariomws) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:217]:
```suggestion
The following updates are applicable for organizations having TeamsOnly users that use Microsoft Teams PowerShell version 5.0.0 or later (note that these changes are only rolled out in commercial environments at present):
```

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 3]
```diff
@@ -541,7 +549,7 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 
 **Updates in Teams PowerShell Module version 5.0.0**
 
-The following updates are applicable for organizations having TeamsOnly users that use Microsoft Teams PowerShell version 5.0.0  (note that these changes are only rolled out in commercial environments at present):
+The following updates are applicable for organizations having TeamsOnly users that use Microsoft Teams PowerShell version >=5.0.0 and will be applicable to all Teams PowerShell Module versions by 15th April 2023  (note that these changes are only rolled out in commercial environments at present):
```
Comment (dariomws) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:552]:
```suggestion
The following updates are applicable for organizations having TeamsOnly users that use Microsoft Teams PowerShell version 5.0.0 or later (note that these changes are only rolled out in commercial environments at present):
```

**Review by dariomws — CHANGES_REQUESTED** [review]

**Review by dariomws — APPROVED** [review]

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 4]
```diff
@@ -211,9 +212,9 @@ Enables you to limit the returned data by filtering on specific attributes. For
 
 The Filter parameter uses the same filtering syntax as the Where-Object cmdlet. For example, the following filter returns only users who have been enabled for Enterprise Voice: `-Filter 'EnterpriseVoiceEnabled -eq $True'` or ``-Filter "EnterpriseVoiceEnabled -eq `$True"``.
 
-**Updates in Teams PowerShell Module version 5.0.0**
+**Updates in Teams PowerShell Module version 5.0.0 and above**
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:215]:
```suggestion
**Updates in Teams PowerShell Module version 5.0.0 and later**
```

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 5]
```diff
@@ -276,7 +277,7 @@ These filtering operators have been reintroduced:
 
 **Updates in Teams PowerShell Module version 3.0.0 and above**
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:278]:
```suggestion
**Updates in Teams PowerShell Module version 3.0.0 and later**
```

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 6]
```diff
@@ -497,6 +498,13 @@ Accept pipeline input: False
...
 
+### -Sort
+
+Sorting is now enabled in Teams PowerShell Module 5.1.0 and above by using the "-Sort" or "-OrderBy" parameters. For Example:
+
+- Get-CsOnlineUser -Filter {LineURI -like *123*} -OrderBy "DisplayName asc"
+- Get-CsOnlineUser -Filter {DisplayName -like '*abc'} -OrderBy {DisplayName desc}
+
 ### -UnassignedUser
 This parameter has been deprecated from the Teams PowerShell Modules version 3.0 or later due to limited usage.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:509]:
```suggestion
This parameter has been deprecated from the Teams PowerShell Modules version 3.0 and later due to limited usage.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @isrumnon, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@get-itips

**Review by get-itips — APPROVED** [review]

**Comment by isrumnon** [comment]
@yogkumgit can we merge this today?

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Comment by isrumnon** [comment]
@tiburd can you please merge this asap?

**Comment by isrumnon** [comment]
@yogkumgit can we please ensure this get merged by today?

**Pull request merged**